patches/@kubernetes+client-node+0.11.1.patch

```diff
@@ -1,24 +1,16 @@
 diff --git a/node_modules/@kubernetes/client-node/dist/oidc_auth.js b/node_modules/@kubernetes/client-node/dist/oidc_auth.js
-index a393b8d..652ce12 100644
+index bbc69a4..e232791 100644
 --- a/node_modules/@kubernetes/client-node/dist/oidc_auth.js
 +++ b/node_modules/@kubernetes/client-node/dist/oidc_auth.js
-@@ -38,6 +38,7 @@ class OpenIDConnectAuth {
-                 return null;
-             }
-             const client = overrideClient ? overrideClient : yield this.getClient(user);
-+            client.CLOCK_TOLERANCE = 10;
-             return this.refresh(user, client);
-         });
-     }
-@@ -46,6 +47,7 @@ class OpenIDConnectAuth {
-             if (Date.now() / 1000 > this.currentTokenExpiration) {
+@@ -76,6 +76,7 @@ class OpenIDConnectAuth {
```

Comment:
Patch is not needed anymore because lens is using separate auth proxy.

Reply:
Yes. I was about to ask if that's still needed. Will remove it.